backend/src/main/kotlin/cz/loono/backend/db/model/Category.kt

```diff
@@ -0,0 +1,19 @@
+package cz.loono.backend.db.model
+
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.ManyToMany
+import javax.persistence.Table
+
+@Entity
+@Table(name = "\"category\"")
+data class Category(
```

Comment:
`Category` is too generic. I would like this to be something like `HealthcareCategory`. We may have more entities with a "category" in the future.